client/src/components/relatedProducts/outfitList.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import Carousel from 'react-multi-carousel';
+import axios from 'axios';
+import API_KEY from '../../../../config.js'
+import Card from './card.jsx'
+// import styled from 'styled-components';
+import "react-multi-carousel/lib/styles.css";
+import AddOutfit from './addOutfitCard.jsx'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faStar } from '@fortawesome/free-solid-svg-icons'
+
+const responsive = {
+  desktop: {
+    breakpoint: { max: 3000, min: 1024 },
+    items: 3.25,
+    slidesToSlide: .75
+  }
+};
+
+const Outfit = (props) => {
+  const [productData, setProductData] = React.useState([]);
+  const [list, setList] = React.useState([<AddOutfit key = {0} handleClick = {()=>{console.log('test add button')}}/>])
+
+  //WILL RECIEVE FROM PROPS
+  const product_id = "37314"
+  const product_style = 0
+
+  React.useEffect(() => {
```

Comment:
This looks pretty complicated but looks like you handled the promise chaining well, nice work.